Readme.md

```diff
@@ -0,0 +1,181 @@
+
+## Deploying
+
+### Deploying Azure Front Door Standard
+
+Use the following command to deploy the Azure Front Door Standard deployment
+
+```bash
+az deployment sub create --template-file deployRgFrontDoorAndWeb.bicep \
+  --location centralus \
+  -p frontDoorSkuName=Standard_AzureFrontDoor assetPrefix=uniqueprefixname
+```
+
+### Deploying Azure Front Door Premium With Private Endpoints
+
+Use the following command to deploy the Azure Front Door Premium deployment
+
+```bash
+az deployment sub create --template-file deployRgFrontDoorAndWeb.bicep \
+  --location centralus \
+  -p frontDoorSkuName=Premium_AzureFrontDoor assetPrefix=uniqueprefixname
+```
+
+### Deploying the VNet for the Front Door Premium deployment
+
+You will need to update the following parameters in parameters.json:
+
+- jumpboxPublicSshKey - This is the public SSH key you created earlier
+- assetPrefix - This is used as the basis for the names of every asset deployed. For example if the assetPrefix is 'testworkload', the Azure Resource Group name will be 'testworkload-rg'. This assetPrefix MUST match the assetPrefix used in the Front Door Premium deployment
+- storageAccountWebsiteLocations - This is an array of valid Azure locations where Storage Accounts will be created.
+- vmadmin - The name of the admin account to create for the Jumpbox Virtual Machine.
+- principalId - The object id of the Service Principal created above. Do not use the appId.
+
+The following is an example of storageAccountWebsiteLocations:
+
+```bash
+"storageAccountWebsiteLocations": {
+    "value": [
+    "eastus",
+    "westus3"
+    ]
+},
+```
+
+Use the following command to deploy the Virtual Network deployment
+
+```bash
+az deployment group create \
+  --template-file deployVnet.bicep \
+  --resource-group resourceGroupName \
+  --parameters @parameters.json 
+```
+
+Note: The resourceGroupName should be the name of the resource group deployed in the Front Door Premium deployment.
+Note: You MUST make sure that the assetPrefix matches the assetPrefix used in the Front Door Premium deployment.
+
+## Adding index.html files to the websites
+
+This repository has 2 index.html files you can use. One is for West US and the other for East US.
+
+Use the following command to log into the Azure CLI as the Service Principal:
+
+```bash
+read -sp "Password for Service Principal: " SP_SECRET && echo && read -sp "appId for Service Principal: " SP_APPID && echo && read -sp "Tenant id for Service Principal: " SP_TENANT && echo && az login --service-principal -u $SP_APPID -p $SP_SECRET --tenant $SP_TENANT
+```
+
+Use the following commands to copy the sample web pages to your storage accounts:
+
+```bash
+az storage blob copy start \
+  --account-name nameOfWestStorageAccount \
+  --destination-blob index.html \
+  --auth-mode login \
+  --destination-container web \
+  --source-uri https://raw.githubusercontent.com/RobBagby/network-secure-ingress-sample/main/sample-websites/west/index.html

```

Comment:
Since the content is so small, and you'll want to loop over their regions, you could just include the content as param of `--data` and that would give you the ability to dynamically inject the storage account name into the body.